app/services/decisions/income_decision_tree.rb

```diff
@@ -163,7 +173,7 @@ def determine_showing_no_income_page
     end
 
     def employed?
-      crime_application.income.employment_status.include?(EmploymentStatus::EMPLOYED.to_s)
+      crime_application.income.employment_status&.include?(EmploymentStatus::EMPLOYED.to_s)
```

Comment:
@RoseSAK Cant think of a scenario where `crime_application.income.employment_status` returns `nil` when income is present.

What do you think of using `!!crime_application.income.employment_status&.include?(EmploymentStatus::EMPLOYED.to_s)`  to return always boolean instead of `nil`?

Reply:
Sure, I'll update this!

Reply:
This is also continuing into the employment journey if the applicant is Employed AND Self-employed - and combination what includes self employed needs to show the eForms exit page for now.